package.json

```diff
@@ -611,6 +611,16 @@
           "default": true,
           "description": "Adds a space after a separator (',' and ';')."
         },
+        "powershell.codeFormatting.WhitespaceInsideBrace": {
+          "type": "boolean",
+          "default": true,
+          "description": "Adds a space after an opening brace ('}') and before a closing brace ('}')."
+        },
+        "powershell.codeFormatting.WhitespaceAroundPipe": {
+          "type": "boolean",
+          "default": true,
+          "description": "Adds a space before and after a pipeline ('|')."
```

Comment:
I think we should refer to this symbol as it is referred to in the docs "pipeline operator" e.g. "before and after the pipeline operator ('|').".